generators/Reporting.fs

```diff
@@ -0,0 +1,92 @@
+module Generators.Reporting
+
+open Serilog
+open CanonicalData
+open Exercise
+open Options
+       
+let private isOutdated (exercise: GeneratorExercise) options (parseCanonicalData':string -> CanonicalData) =
+    let cData = parseCanonicalData' exercise.Name
+    match cData.Version,exercise.ReadVersion() with
+    | canonVersion,exerciseVersion when canonVersion.Equals exerciseVersion -> false
+    | _,_ -> true
```

Comment:
I think you can replace this with just a single underscore, there is no need to pattern match on the tuple's contents.